src/main/app/access-code/AccessCodePostController.ts

```diff
@@ -25,29 +25,37 @@ export class AccessCodePostController {
 
     const { saveAndSignOut, saveBeforeSessionTimeout, _csrf, ...formData } = form.getParsedBody(req.body);
 
-    formData.respondentUserId = req.session.user.id;
-    formData.applicant2Email = req.session.user.email;
+    //ToDo: confirm setting the first and last names from session is
```

Comment:
Have you looked into this already?

Reply:
Yes. This was a comment on Del's original PR on the subject. Will need to remove the comment. Will do so.

Reply:
I was just looking  at why this was done as part of NFDIV-2166 (only for app2 in joint application) and it does need a little bit of thought additionally.